dom/schema/v1/tollgate_crossing.sql

```diff
@@ -0,0 +1,8 @@
+create table if not exists tollgate_crossing
+(
+    id          text        not null primary key,
+    tollgate_id text        not null,
+    driver_id   text        not null,
```

Comment:
We have to either use `worker` everywhere or `driver`. I thought we use `worker`. And the `_id` suffix is redundant as well, in my very humble opinion )

Reply:
thought about worker vs driver too, and decided to use driver as tollgate is crossed by drivers. But have no strong opinion on this. We can refactor. 
for _id suffixes particular case i'd still used full naming tollgate_id to avoid ambiguity. Explicit vs implicit. it is visually more clear what relations the table has. otherwise when many columns used, it is hard to visually grasp all '_id' columns.
 
But it is nothing more than convention.

Reply:
Let's @jasonprado approve use of driver vs worker. Later we have to remember where to use worker vs. driver when writing SQL queries. I don't like it.

Reply:
i renamed driver to worker everywhere. let it be worker from now on.